### ISSUE TITLE
BUG: Row count is wrong when table filter is on

### ISSUE THREAD

**Issue by idangozlan** [beekeeper-studio/beekeeper-studio#1179]
**Describe the bug**
The total number of rows is not getting updated when filtering the results

**To Reproduce**
1. Go to a table with more than 1 record
2. Filter so you'll have 1 record match
3. See the total number of rows in the left bottom
4. See the total number is representing the total rows count instead of the total filtered rows count

**Expected behavior**
Get the total filtered rows count

**Version Information (please complete the following information):**
 - OS: Mac
 - App Version 3.4.1 (3.4.1)
 - Database type: Postgres

**Comment by rathboma** [comment]
Yes it is. I've disabled filtered row counts because they cause massive performance problems for users with large datasets.

I'll keep this ticket -- it should probably just replace the count with `--` and add a way to load it with a click.
